backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java

```diff
@@ -6,7 +6,8 @@
 
 public enum OrganizationRole {
     ENTRY_ONLY("Test-entry users",
-            EnumSet.of(UserPermission.START_TEST, UserPermission.UPDATE_TEST, UserPermission.SUBMIT_TEST)),
+            EnumSet.of(UserPermission.READ_PATIENT_LIST, UserPermission.START_TEST, UserPermission.UPDATE_TEST, 
```

Comment:
Per discussion elsewhere, I will be amending this to `SEARCH_PATIENT_LIST` so we don't confuse the front end.